src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,50 @@
 import React from 'react';
-
 import './MovieCard.scss';
+import { Movie } from '../../types/Movie';
+
+type Props = {
```

Comment:
```suggestion
interface Props = {
```
Change `Props` to something more descriptive